ansible/roles/request_ocp_fyre/defaults/main.yml

```diff
@@ -9,7 +9,8 @@ fyre_ocp_inf_group: "ocp-clusters"
 fyre_inf_node_name: "{{ clusterName | lower }}-inf"
 fyre_site: "svl"
 fyre_ocptype: "ocp"
-ssh_public_key_file: "{{ lookup('file', lookup('env','HOME') + '/.ssh/id_rsa.pub') }}" 
```

Comment:
Im curious why did you change from this default?

Reply:
oh, as I tested I thought it was just adding the filename into the json, and not the actual content.  Maybe I was misusing the variable.   I did rename to ssh_public_key to make it more about the key value and not the filename.